dags/tasks/folio/request.py

```diff
@@ -7,22 +7,27 @@ def FolioRequest(**kwargs) -> SimpleHttpOperator:
     conn_id = kwargs.get("conn_id")
     dag = kwargs.get("dag")
     token = kwargs.get("x-okapi-token")
```

Comment:
not changed by this PR, but should this be `token = kwargs.get("token")`?

Reply:
Yes, thanks! Changed in latest commit.